filesystems/localfilesystem/src/main/kotlin/com/lightteam/localfilesystem/utils/Extensions.kt

```diff
@@ -39,16 +39,25 @@ fun File.size(): Long {
     return length()
 }
 
-fun Long.formatAsDate(): String {
-    val dateFormat = SimpleDateFormat("dd/MM/yyyy EEE HH:mm", Locale.getDefault())
+// TODO: 2020/8/5 Require localized
+fun Long.formatAsDate(pattern: String): String {
+    val dateFormat = SimpleDateFormat(pattern, Locale.getDefault())
     return dateFormat.format(this)
 }
 
+@Deprecated(
+    "Require localized, such as format is `yyyy/MM/dd EEE HH:mm` in China",
+    replaceWith = ReplaceWith("formatAsDate(pattern)")
+)
+fun Long.formatAsDate(): String {
+    return this.formatAsDate("dd/MM/yyyy EEE HH:mm")
```

Comment:
`this` is unnecessary

```suggestion
    return formatAsDate("dd/MM/yyyy EEE HH:mm")
```